package.json

```diff
@@ -17,6 +17,9 @@
     "not ie <= 9"
   ],
   "main": "./target/js/okta-sign-in.js",
+  "engines": {
+    "node": ">=12.22"
```

Comment:
Any reason to use this specific version?

Reply:
Yes, that is the minimum required by the HTML validate module